src/llmq/quorums_dkgsession.cpp

```diff
@@ -1200,6 +1252,10 @@ void CDKGSession::MarkBadMember(size_t idx)
 {
     auto member = members.at(idx).get();
     if (member->bad) {
+        quorumDKGDebugManager->UpdateLocalMemberStatus(params.type, idx, [&](CDKGDebugMemberStatus& status) {
+            status.bad = true;
+            return true;
+        });
```

Comment:
We fixed this in another branch, this block of code shouldn't be here. We are updating the status below and only once.

Reply:
Yep. forgot to include that commit. fixed